system/README.md

```diff
@@ -3,15 +3,15 @@
 System tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of load tests. 
 
 ## Related repositories: 
-- System tests that run both on AdoptOpenJDK and AdoptOpenJDK-OpenJ9 SDKs are located at:  github.com/AdoptOpenJDK/openjdk-systemtest.git
+- System tests that run both on AdoptOpenJDK and AdoptOpenJDK-OpenJ9 SDKs are located at:  github.com/AdoptOpenJDK/aqa-systemtest.git
```

Comment:
AdoptOpenJDK -> adoptium

Reply:
Fixed. I had missed a few more actually! Updated the PR with them too.